### PR TITLE
Fixed gol_lazar_belta literate file

### PR DESCRIPTION
**Pull request by egidioln**
Fixed bugs in the literate file for gol lazar belta example

### CODE REVIEW

File: src/mpc.jl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module MPC
+
+mutable struct Optimizer <: MOI.AbstractOptimizer
+end
+
+function MOI.optimize!(optimizer::Optimizer)
```
Comment (blegat) [inline src/mpc.jl:6]: I guess you didn't mean to push this

File: docs/src/examples/Gol, Lazar & Belta (2013).jl [hunk 2]
```diff
@@ -160,8 +161,6 @@ x2 = [xu.x[j][2] for j in eachindex(xu.x)]
...
 # ### References
-# <a id="1">[1]</a> 
-# Gol, E. A., Lazar, M., & Belta, C. (2013). Language-guided controller synthesis for linear systems. IEEE Transactions on Automatic Control, 59(5), 1163-1176.
-#
-# <a id="2">[2]</a>
-# Bemporad, A., & Morari, M. (1999). Control of systems integrating logic, dynamics, and constraints. Automatica, 35(3), 407-427.
\ No newline at end of file
+# 
+# 1. Gol, E. A., Lazar, M., & Belta, C. (2013). Language-guided controller synthesis for linear systems. IEEE Transactions on Automatic Control, 59(5), 1163-1176.
+# 1. Bemporad, A., & Morari, M. (1999). Control of systems integrating logic, dynamics, and constraints. Automatica, 35(3), 407-427.
```
Comment (blegat) [inline docs/src/examples/Gol, Lazar & Belta (2013).jl:166]: Shouldn't this 1. be 2.?

File: docs/src/examples/Gol, Lazar & Belta (2013).jl [hunk 3]
```diff
@@ -56,15 +54,15 @@ transition_cost = QuadraticControlFunction(ones(1, 1))
 problem = OptimalControlProblem(
     system,
     q0, x0,
-    Fill(state_cost, N),
-    Fill(Fill(transition_cost, ntransitions(system)), N),
+    Fill(state_cost, N),
```
Comment (blegat) [inline docs/src/examples/Gol, Lazar & Belta (2013).jl:57]: Remove trailing space at the end of the line

File: docs/src/examples/Gol, Lazar & Belta (2013).jl (hunk 2 above)
Comment (egidioln) [reply docs/src/examples/Gol, Lazar & Belta (2013).jl:166]: It updates to the correct numbers when the .md is generated
image

File: src/mpc.jl (hunk 1 above)
Comment (egidioln) [reply src/mpc.jl:6]: No idea where this came from... I'll fix it

File: docs/src/examples/Gol, Lazar & Belta (2013).jl (hunk 3 above)
Comment (egidioln) [reply docs/src/examples/Gol, Lazar & Belta (2013).jl:57]: Sure!

**Pull request merged**